src/llama_stack_client/lib/inference/event_logger.py

```diff
@@ -21,13 +21,22 @@ def print(self, flush=True):
         cprint(f"{self.content}", color=self.color, end=self.end, flush=flush)
 
 
+class InferenceStreamLogEventLogger:
+    def process_chunk(self, chunk):
+        event = chunk.event
+        if event.event_type == "start":
+            return InferenceStreamLogEvent("Assistant> ", color="cyan", end="")
+        elif event.event_type == "progress":
+            return InferenceStreamLogEvent(event.delta.text, color="yellow", end="")
+        elif event.event_type == "complete":
+            return InferenceStreamLogEvent("")
+        return None
+
+
 class EventLogger:
```

Comment:
why do we need these EventLogger classes?

Reply:
@raghotham it is a useful utility for printing out (however they have been overused and misnamed)